Description: Fix unicode string in Python 3.2
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-07-13

Index: python-oslo.i18n/tests/test_public_api.py
===================================================================
--- python-oslo.i18n.orig/tests/test_public_api.py	2015-02-07 21:09:45.000000000 +0100
+++ python-oslo.i18n/tests/test_public_api.py	2015-02-07 21:16:04.000000000 +0100
@@ -16,6 +16,8 @@
 
 import mock
 
+import six
+
 from oslo import i18n
 from oslo_i18n import _lazy
 
@@ -41,4 +43,4 @@
             i18n.enable_lazy(original)
 
     def test_translate(self):
-        i18n.translate(u'string')
+        i18n.translate(six.u('string'))
