Description: Fix a python 3.2 unicode
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2014-09-05

--- python-oslo.i18n-0.2.0.orig/tests/test_fixture.py
+++ python-oslo.i18n-0.2.0/tests/test_fixture.py
@@ -35,4 +35,4 @@ class FixtureTest(test_base.BaseTestCase
         # Python 2.6 does not have assertNotIsInstance
         self.assertFalse(isinstance(msg, _message.Message))
         self.assertIsInstance(msg, six.text_type)
-        self.assertEqual(msg, u'this is a lazy message')
+        self.assertEqual(msg, six.u('this is a lazy message'))
